sdk/create-dapp/src/index.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable no-restricted-globals */
+
+import { existsSync, statSync } from 'fs';
+import { mkdir, readdir, readFile, writeFile } from 'fs/promises';
+import { relative, resolve } from 'path';
+import { prompt } from 'enquirer';
+
+async function main() {
+	const results = await prompt<{
+		template: string;
+		dAppName: string;
+	}>([
+		{
+			type: 'select',
+			name: 'template',
+			message: 'Select a template',
+			choices: [
```

Comment:
As someone reading the choices & pretending I don't have all the context, I'm not sure if I'd immediately be able to differentiate between a "client dApp" and an "end to end" dApp.

I don't know if I have better naming suggestions 😆, but maybe more descriptive messages here would help? The client dApp is just an app that reads on-chain data and the end-to-end dApp is an app that interacts and performs actions with smart contracts, but I could see people potentially confusing client/end-to-end with stuff like Next.js, some non-smart contract backend, etc.